File: PHP/Example Statement/Example Switch Case/Switch Case.php
<!DOCTYPE HTML>
<html>

<head>
	<title>Example Switch Case</title>
</head>

<body>

	<?php
	$varchc = "";

	if ($_SERVER["REQUEST_METHOD"] == "POST") {
		$varchc = test_input($_POST["varchc"]);
	}

	function test_input($data)
	{
		$data = trim($data);
		$data = stripslashes($data);
		$data = htmlspecialchars($data);
		return $data;
	}
	?>

	<tr>
		<td>Menu :</td>
		<td><br><br></td>
		<td>1. First Example</td>
		<td><br></td>
		<td>2. Second Example</td>
		<td><br></td>
		<td>3. Third Example</td>
		<td><br></td>
		<td>4. Fourth Example</td>
		<td><br></td>
		<td>Enter your Choice : </td>
		<td><br><br></td>
	</tr>

	<form method="post" action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]); ?>">
		Enter your Choice : <input type="text" name="varchc">
		<br>
		<br>
		<input type="submit" name="submit" value="Submit">
	</form>

	<?php
	switch ($varchc) {
		case "1":
			echo "<br>Example 1st Succeed";
			break;
		case "2":
			echo "<br>Example 2nd Succeed";
			break;
		case "3":
			echo "<br>Example 3rd Succeed";
			break;
		case "4":
			echo "<br>Example 4th Succeed";
			break;
		default:
			echo "Enter your Choice Correctly";
	}
	?>

</body>

</html>

File: C++/Example Statement/Example Switch Case/Switch Case.cpp
#include <iostream>

using namespace std;

int main() {
	
int varchc;
    
    cout << "Menu :\n" << endl;
    cout << "1. First Example           " << endl;
    cout << "2. Second Example          " << endl;
    cout << "3. Third Example           " << endl;
    cout << "4. Fourth Example        \n" << endl;
    cout << "Enter your Choice : "; 
    cin >> varchc;
    switch (varchc)
    {
        case 1: {
			cout << "Example 1st Succeed" << endl;
       	    break;
    	}
		case 2: {
            cout << "Example 2nd Succeed" << endl;
            break;
       	}
      	case 3: {
        	cout << "Example 3rd Succeed" << endl;
            break;
        }
        case 4: {
        	cout << "Example 4th Succeed" << endl;
        	break;
        }
    	default: {
            cout << "Enter your Choice Correctly" << endl;
            break;
        }
    }
 
 	return 0;
 
}